org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/dom/ASTConverter_23Test.java

```diff
@@ -133,4 +127,64 @@ void main() {
 		assertEquals("Not a Block", block.getNodeType(), ASTNode.BLOCK);
 		assertEquals("Block startPosition is not correct", block.getStartPosition(), 21);
 	}
+
```

Comment:
Would help if you can select just the newly added code block and format, to keep it in line with our formatting preferences.